translations/ja.json

```
@@ -454,57 +454,57 @@
+	"Profile name '{{.name}}' is duplicated with machine name '{{.machine}}' in profile '{{.profile}}'": "プロファイル名 '{{.name}}' は '{{.profile}}' プロファイル中のマシン名 '{{.machine}}' と重複しています",
+	"Profile name '{{.name}}' is not valid": "プロファイル名 '{{.name}}' は無効です",
+	"Profile name '{{.profilename}}' is not valid": "プロファイル名 '{{.profilename}}' は無効です",
+	"Profile name should be unique": "プロファイル名は単一でなければなりません",
+	"Provide VM UUID to restore MAC address (hyperkit driver only)": "MAC アドレスを復元するための VM UUID を指定します (hyperkit ドライバーのみ)",
+	"Pull images": "イメージをプルします",
+	"Pull the remote image (no caching)": "リモートイメージをプルします (キャッシュなし)",
 	"Pulling base image ...": "イメージを Pull しています...",
-	"Push images": "",
-	"Push the new image (requires tag)": "",
-	"Reboot to complete VirtualBox installation, verify that VirtualBox is not blocked by your system, and/or use another hypervisor": "",
-	"Rebuild libvirt with virt-network support": "",
-	"Received {{.name}} signal": "",
-	"Recreate the cluster by running:\n\t\tminikube delete {{.profileArg}}\n\t\tminikube start {{.profileArg}}": "",
-	"Registries used by this addon. Separated by commas.": "",
-	"Registry addon with {{.driver}} driver uses port {{.port}} please use that instead of default port 5000": "",
-	"Registry mirrors to pass to the Docker daemon": "Docker デーモンに渡すレジストリ ミラー",
-	"Reinstall VirtualBox and reboot. Alternatively, try the kvm2 driver: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/": "",
-	"Reinstall VirtualBox and verify that it is not blocked: System Preferences -\u003e Security \u0026 Privacy -\u003e General -\u003e Some system software was blocked from loading": "",
-	"Related issue: {{.url}}": "",
-	"Related issues:": "",
+	"Push images": "イメージをプッシュします",
+	"Push the new image (requires tag)": "新イメージをプッシュします (タグが必要)",
+	"Reboot to complete VirtualBox installation, verify that VirtualBox is not blocked by your system, and/or use another hypervisor": "VirtualBox インストールを完了させるために再起動し、VirtualBox がシステムや別のハイパーバイザーにブロックされていないことを検証します",
+	"Rebuild libvirt with virt-network support": "virt-network サポート付きで libvirt を再構築します",
+	"Received {{.name}} signal": "{{.name}} シグナルを受信しました。",
+	"Recreate the cluster by running:\n\t\tminikube delete {{.profileArg}}\n\t\tminikube start {{.profileArg}}": "次のコマンドを実行してクラスタを再作成してください:\n\t\tminikube delete {{.profileArg}}\n\t\tminikube start {{.profileArg}}",
+	"Registries used by this addon. Separated by commas.": "このアドオンで使用するレジストリー。コンマで区切ります。",
+	"Registry addon with {{.driver}} driver uses port {{.port}} please use that instead of default port 5000": "{{.driver}} ドライバーを使うレジストリーアドオンは {{.port}} 番ポートを使用します。デフォルトの 5000 番ポートの代わりにこちらのポートを使用してください",
+	"Registry mirrors to pass to the Docker daemon": "Docker デーモンに渡すミラーレジストリー",
+	"Reinstall VirtualBox and reboot. Alternatively, try the kvm2 driver: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/": "VirtualBox を再インストールして再起動します。あるいは、kvm2 ドライバーを試してください: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/",
```

```suggestion
	"Reinstall VirtualBox and reboot. Alternatively, try the kvm2 driver: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/": "VirtualBox を再インストールして再起動してください。あるいは、kvm2 ドライバーを試してください: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/",
```
Advice.
https://github.com/kubernetes/minikube/blob/a493b59116c3707102372537c2159a9615cc16b1/pkg/minikube/reason/known_issues.go#L518

---

Ditto.